### ISSUE TITLE
"EFI_NOT_FOUND" after "Booting NT Version..."

### ISSUE THREAD

**Issue by 25pwn** [maharmstone/quibble#61]
I tried to load Windows 10 Pro Workstation 21H2 from a btrfs subvol, and got:
```
ARC name is btrfs(4c6d4155-eb9a-4924-93a0-f4d8a179631d).
Loaded ntoskrnl.exe at fffff80800000000.
Booting NT version 10.0.19041.1288.
hive->FindKey returned EFI_NOT_FOUND
loadkey_registry returned EFI_NOT_FOUND
boot returned EFI_NOT_FOUND
```
`freeldr.ini`:
```
[FREELOADER]
TimeOut=10
DefaultOS=Windows

[Operating Systems]
Windows="Windows"
Windows_Debug="Windows (Debug)"

[Windows]
SystemPath=btrfs(4c6d4155-eb9a-4924-93a0-f4d8a179631d)\Windows
Options=/SUBVOL=101
[Windows_Debug]
SystemPath=btrfs(4c6d4155-eb9a-4924-93a0-f4d8a179631d)\Windows
Options=/SUBVOL=101 /DEBUG
```
Please note that I installed the OS in an unusual way.
First, I used dism to install winbtrfs to Windows PE and the target wim file. I confirmed it was installed to WinPE by running `mkbtrfs`, which was present. btrfs filesystems were automounted. I could use `dir` to read from them, and the wim file I tried to apply was also on a btrfs filesystem.
However, diskpart showed the partition as RAW and not mounted, and refused to mount it. Attempting to apply the image with dism gave me `Error: 5 Access is denied.`
So under linux, I mounted the wim file using wimlib's wimmount and used `cp -r --preserve=all` to copy its contents to the subvolume.
Then, attempting to boot it via quibble gave me the above errors.
Running bcdboot for the install under WinPE gave me the same results.
The /debug flag also gave me the same results.

**Comment by adamantinum** [comment]
I did the same thing. I think it depend on the Windows version, using a Windows 7 wim I'm able to reach the "Booting Windows..." part, but then it stucks.

**Comment by i2** [comment]
Yes, I have the same issue and I have tried on a bunch of computers to no avail.
